### ISSUE TITLE
/api/vehicles returns 0 counts

### ISSUE THREAD

**Issue by markwj** [openvehicles/Open-Vehicle-Server#9]
```
https://api.openvehicles.com:6869/api/vehicles and I get a response:
[
    {
        "id": "<my vehicle id>",
        "v_apps_connected": 0,
        "v_btcs_connected": 0,
        "v_net_connected": 0
    }
]
```

The counts do not seem accurate.

**Comment by sdarveau** [comment]
I'm a developer so I'd like to help with this. I've done my setup and started the server but I get
error OVMS::Server::Plugin: - - - Function DbGetVehicle does not exist

I get this message for all database functions (Db*). I'm on MacOS and not familiar with perl. What am I missing?

**Comment by markwj** [comment]
Resolved in 3.0.001-16-gd98deef.
